bread/layout/components/search.py

```diff
@@ -105,9 +105,12 @@ def _loading_indicator(resultcontainerid):
     )
 
 
-def _close_button(resultcontainerid):
+def _close_button(resultcontainerid, widgetattributes):
     kwargs = {
-        "_class": "bx--search-close bx--search-close--hidden",
+        "_class": hg.BaseElement(
+            "bx--search-close",
+            hg.If(widgetattributes.get("value"), None, " bx--search-close--hidden"),
+        ),
```

Comment:
This should be here anyway, was missing. In order to show the close-button always if there is some value inside the search bar.

Reply:
how does `None` get rendered?

Reply:
```htmlgenerator``` will ignore ```None``` elements when rendering. This is a little "trick" because we have no lazy ```not``` operator, so we simply switch the if and the else parts.